modules/main-rules/pages/leadership-test.adoc

```diff
@@ -4,12 +4,15 @@ When your detachment has one or more Blast markers on it, you must pass a Leader
 
 {blank}[TODO: What happens if you try to set Overwatch orders and fail? Default to Advance orders I presume, and lose the chance to use any other orders? If you pass the test to *set* Overwatch orders, do you get to *move* up to 5 cm under those orders without a further test?]
 
-NOTE: A Broken detachment is not under orders and no Leadership test is required to make a Retreat move (see xref:broken-detachments.adoc[]).
+NOTE: A detachment that is subject to the Broken condition is not under orders -- so, a Retreat move does not trigger a Leadership test.
+{blank}[TODO: There's at least one special case to consider — the {hit-and-run} ability's use of the Retreat move' Should this ability require a Leadership test if the detachment has Blast markers, like a normal move would? If yes then it'd be best to adjust this note to accomodate it better, or possibly change the {hit-and-run} text a bit.]
```

Comment:
'can always' in the original text, for me, infers no requirement for a Leadership test.

And it would also cause extra problems - what happens if they choose to make a retreat move and fail the Leadership test? They've chosen to make a retreat move, so everyone within 15 cm of an enemy would die. :-(

Reply:
That seems pretty clear now, I guess I was over-thinking it in the moment of editing.

Good point — not only would a Leadership test for the _Hit and Run_ case clash with the NOTE here and require a special exception, there's the problem you've described.

I'm happy to declare my TODO note totally redundant in this case and resolve this thread. 😅

```suggestion
```

Reply:
Sounds good to me.